apps/stack/stacks/TransformStack.ts

```diff
@@ -29,4 +29,22 @@ export function TransformStack({ stack }: StackContext) {
     }
   });
 
+  ExtractBus.addRules(stack, {
+    "transformMergeRequests": {
+      pattern: {
+        source: ["extract"],
+        detailType: ["mergeRequest"]
+      },
+      targets: {
+        transformMergeRequests: {
+          function: {
+            bind: [TRANSFORM_DB_URL, TRANSFORM_DB_AUTH_TOKEN],
```

Comment:
TRANSFORM_DB_AUTH_TOKEN -> TRANSFORM_DB_TOKEN
Da bi bilo koegzistentno sa migrate-dev.ts skriptom radi lakseg migriranja baze

Reply:
@ivke995 It's already defined in turbo.json to be 

`"EXTRACT_DATABASE_URL",
    "EXTRACT_DATABASE_AUTH_TOKEN",
    "TRANSFORM_DATABASE_URL",
    "TRANSFORM_DATABASE_AUTH_TOKEN",`

Reply:
@ivke995 
Yh we should fix this in the migration script, I didn't notice the turbo config env vars.

In regards to sst config, I'm pro renaming it, but it will take a sync step for everyone to fix their stack config. Since we should rename our sst app at some point, we can do it then since that requires a sync step too